docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+---
+title:  jpsreport inifile
+keywords: analysis
+tags: [jpsreport, file]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: In the configuration file, all important information needed for the analysis are defined. For example the location of the trajectory and geometry files, the measurement methods to use, etc.
+permalink: jpsreport_inifile.html
+last_updated: Dec 20, 2019
+---
+
+
+
+In the configuration file, the following sections should be defined:
+
+## Header
+
+```xml
+  <?xml version="1.0" encoding="UTF-8" ?>
+  <JPSreport project="JPS-Project" version="0.8" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://github.com/JuPedSim/jpsreport/blob/develop/xsd/jps_report.xsd">
+```
+
+## Logfile
+It might be useful to define a logfile (for debugging purposes):
+```xml
+    <logfile>log.txt</logfile>
+```
+
+## Geometry
+indicates the file name  corresponding to the trajectory files to analyze.
+
+```xml
+   <geometry file = "geo_KO_240_050_240.xml" />
+```
+
+The location can be either absolute path or relative path to the location of the inifile. A path is considered absolute if it starts with "/" (Linux system) or contains ":" (Windows system).
+
+```xml
+   <geometry file = "geo/geo_KO_240_050_240.xml" />
+```
+
+## Output
+indicates the location of the output files based on the location of inifile or the absolute path.
+
+```xml
+   <output location="Output/"/>
+```
+
+A path is considered absolute if it starts with "/" (Linux system) or contains ":" (Windows system).
+
+## Trajectories
+
+  indicates the location and the name of the trajectory files that will be analyzed.
+  The format of trajectory files should be `.txt` or `.xml`.
+  The supported unit of the trajectories is `m`.
+  Two other sub-options `file` and `path` can be supplied.
+  If only `path` is given, then all files with the corresponding format in
+  the given folder will be considered as the upcoming trajectories
+  and `JPSreport` will try to load them one by one.
+  If both `file ` and `path` are given, then only the given trajectories
+  will be considered (several `file` tags can be given at the same time).
+
+  The location can be either absolute path or relative path to the location of the inifile.
+  A path is considered absolute if it starts with "/" (Linux system) or contains ":" (Windows system).
+
+  For example:
+
+  ```xml
+  <trajectories format="txt" unit="m">
+      <file name="traj_KO_240_050_240.txt" />
+      <file name="traj_KO_240_060_240.txt" />
+      <path location="./" />
+  </trajectories>
+  ```
+
+## Scripts [depricated since 0.8.5]
```

Comment:
The version number must be corrected.

Reply:
What is the correct number?

Reply:
Good question. After the union of jpscore and jpsreport, @zeroset  and you (@chraibi) should clarify this. JPScore is currently at [0.8.4](https://github.com/JuPedSim/jpscore/releases). JPSreport was already at [0.8.5](https://github.com/JuPedSim/jpsreport/releases).

Reply:
@gjaeger I did suggest to @zeroset to make a `0.9` release asap after merging #592. That would be a nice opportunity to "sync" the version numbers.

Reply:
Good suggestion.

Reply:
@chraibi Can you create a new issue after this PR?